src/database/migrations/20240427140502-create-user.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
         type: Sequelize.STRING,
       },
       password: {
+        allowNull: true,
```

Comment:
its better to not edit already existing migration, you would need create another migration to change allow null constraints on existing table